### PR TITLE
1843 Summary page for properties

### LINKED ISSUE #1843: Create a summary page for datasets missing properties
## Background
It's not easy to identify which datasets are missing properties.

## Feature
Create a summary page listing the datasets that are missing _Mandatory_ and _Recommended_ properties, so it's easy to fix them

## Proposed Solution
Some summary page with statistics and listing of datasets having these issues. Ideally with click-through to the dataset.

### PR DESCRIPTION
**Pull request by AdrianOlosutean**
Closes #1843

### CODE REVIEW

**Review by benedeki — COMMENTED** [review]
I really like it. And seems to work as we talked about it.
Small thing to consider:
In the left pane/menu, the caption says _Dataset Properties_. The button and the stats bellow mention it just as _P/properties_. I think it would be better understood to unify all, and _Dataset Properties_ seems to better serve that.

File: menas/ui/components/home/landingPage.view.xml [hunk 1]
```diff
@@ -85,6 +97,20 @@
                   maintainAspectRatio="false">
               </lab:BarChartJS>
             </Panel>
+            <FlexBox height="100px" alignItems="Start" justifyContent="Center" class="sapUiResponsiveMargin">
+                <items>
+                    <VBox class="sapUiSmallMarginEnd">
+                        <Text text="Missing Mandatory Properties"/>
```
Comment (benedeki) [inline menas/ui/components/home/landingPage.view.xml:103]:
```suggestion
                        <Text text="Missing Mandatory Dataset Properties"/>
```

File: menas/ui/components/home/landingPage.view.xml [hunk 2]
```diff
@@ -85,6 +97,20 @@
...
+                        <NumericContent icon="sap-icon://warning" value="{path: '/landingPageInfo/totalNumberMissingMandatoryProperties', formatter: '.tileNumberFormatter'}"></NumericContent>
+                    </VBox>
+                    <VBox class="sapUiSmallMarginEnd">
+                        <Text text="Missing Recommended Properties"/>
```
Comment (benedeki) [inline menas/ui/components/home/landingPage.view.xml:107]:
```suggestion
                        <Text text="Missing Recommended Dataset Properties"/>
```

**Review by benedeki — APPROVED** [review]
- [x] code reviewed
- [x] pulled
- [x] built
- [x] run

**Review by Zejnilovic — APPROVED** [review]
Tested the running web application

**Pull request merged**